pkg/webhook/server/certificate/manager.go

```diff
@@ -57,58 +56,71 @@ func NewManager(
 	client client.Client,
 	webhookName string,
 	webhookType WebhookType,
-) *manager {
-
-	m := &manager{
-		stopCh:           make(chan struct{}),
-		client:           client,
-		webhookName:      webhookName,
-		webhookType:      webhookType,
-		keyPairByService: map[types.NamespacedName]*triple.KeyPair{},
-		now:              time.Now,
-		ready:            false,
+	certsDuration time.Duration,
```

Comment:
s/duration/expiration

Reply:
expiration is a date, this is elapsed time for them to expire meaning the duration of the certificate.